zcash_client_sqlite/src/wallet.rs

```diff
@@ -2020,7 +2020,21 @@ pub(crate) fn block_fully_scanned<P: consensus::Parameters>(
         //
         // The fully-scanned height is therefore the last height that falls within the first range in
         // the scan queue with priority "Scanned".
-        // SQL query problems.
+        let calc_fully_scanned_height = |row: &rusqlite::Row| {
```

Comment:
This change is fine, but for my education what in the MSRV change prompted it?

Reply:
The new lint is "in a `match` scrutinee, avoid complex blocks or closures with blocks; instead, move the block or closure higher and bind it with a `let`".

https://rust-lang.github.io/rust-clippy/master/index.html#blocks_in_conditions